polkadot/runtime/parachains/src/assigner_parachains.rs

```diff
@@ -44,13 +44,13 @@ pub mod pallet {
 	pub trait Config: frame_system::Config + configuration::Config + paras::Config {}
 }
 
-#[derive(Debug, Encode, Decode, TypeInfo)]
+#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Clone)]
 pub struct ParachainsAssignment {
 	pub para_id: ParaId,
 }
 
 impl ParachainsAssignment {
-	fn new(para_id: ParaId) -> Self {
+	pub fn new(para_id: ParaId) -> Self {
```

Comment:
Why does this need to be pub?

Reply:
In some discarded version of this pr, probably. Though not anymore 👍